content/40.inequality.md

```diff
@@ -1,67 +1,125 @@
-The evidence clearly indicates that social environmental factors are critical determinants of individuals' and communities' risks related to COVID-19.
-There are distinct components to COVID-19 susceptibility, and an individual's risk can be elevated at one or all stages from exposure to recovery/mortality: an individual may be more likely to be exposed to the virus, more likely to get infected once exposed, more likely to have serious complications once infected, and be less likely to receive adequate care once they are seriously ill.
-The fact that differences in survival between Black and white patients were no longer significant after controlling for comorbidities and socioeconomic status (type of insurance, neighborhood deprivation score, and hospital where treatment was received) in addition to sex and age [@doi:10.1001/jamanetworkopen.2020.18039] underscores the relevance of social factors to understanding mortality differences between racial and ethnic groups.
-Moreover, the Black patients were younger and more likely to be female than white patients, yet still had a higher mortality rate without correction for the other variables [@doi:10.1001/jamanetworkopen.2020.18039].
-Here, we outline a few systemic reasons that may exacerbate the COVID-19 pandemic in communities of color.
-
-### Factors Observed to be Associated with Susceptibility
+## Social Factors Associated with COVID-19 Exposure and Outcomes and their Biological Repercussions
+
+### Abstract
+
+### Importance
+
+### Introduction
+
+_Severe acute respiratory syndrome-related coronavirus 2_ (SARS-CoV-2) virus has become a serious worldwide threat since late 2019.
+The scientific community has responded by rapidly collecting information and conducting research about the SARS-CoV-2 virus and the associated illness, novel coronavirus disease 2019 (COVID-19).
+The biology of the SARS-CoV-2 virus provides important insight into the spread of the COVID-19 pandemic and the disease's effect on the human body [@individual-pathogenesis].
+Similarly, a thorough understanding of the biology of the human host is also critical to understanding the clinical manifestations of the disease.
+A complete perspective on this disease therefore also requires considering factors that shape variation in human biology, including the environmental factors that are known to influence COVID-19 outcomes.
+
+During the pandemic, a number of risk factors have been associated with COVID-19
+These include health conditions such as obesity, diabetes mellitus, and heart disease [@url:https://www.mayoclinic.org/diseases-conditions/coronavirus/symptoms-causes/syc-20479963; @doi:10.1016/j.jinf.2020.04.021].
+An observed association among racial and ethnic background and COVID-19 outcomes has also led to racial and ethnic minorities frequently being included on the list of groups with increased susceptibility (e.g., [@url:https://www.cdc.gov/coronavirus/2019-ncov/need-extra-precautions/index.html]).
```

Comment:
Something I'm wondering here- this paper discusses COVID-19 on a global scale, but I feel like this sentence is very country specific (because as you outline below, racial and ethnic background on a global scale does not seem to be associated with COVID-19 outcomes, but minority racial status seems to be) Just thinking that you might want to specify that this observation is context-dependent? such as "An observed association in several countries among racial and ethnic background and COVID-19 outcomes has also led to racial and ethnic minorities frequently being included on the list of groups with increased susceptibility"

Reply:
This is a really good point! I will clean this up.